configure.ac

```diff
@@ -11,8 +11,8 @@ AC_PREREQ([2.63])
 define([scm_r], esyscmd([sh -c "git rev-parse --short=7 HEAD"]))
```

Comment:
agree with @miked-mellanox and @shamisp :
1. package VERSION into dist tarball
1. recover from "git rev-parse" failure - if .git or /usr/bin/git not present